src/__tests__/components/common/Toast.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+import Toast from '../../../components/common/Toast/Toast';
+
+let wrapper;
+const props = {
+  show: true,
+};
+jest.useFakeTimers();
+
+describe('<Toast />', () => {
+  beforeEach(() => {
+    wrapper = mount(<Toast {...props} />);
+  });
```

Comment:
@NkFab Can you add an empty line after this line?

Reply:
Done